frontend/src/v5/ui/components/shared/modalsDispatcher/templates/imagesModal/imageMarkup/markupToolbar/buttons/strokeWidthButton/strokeWidthButton.component.tsx

```diff
@@ -25,9 +25,9 @@ import { ToolbarSelect } from '@controls/toolbarSelect/toolbarSelect.component';
 import { Tooltip } from '@mui/material';
 
 const VALUE_TO_WIDTH = invert(STROKE_WIDTH);
-const Icon = ({ value, title }) => (
+const Icon = ({ value, title, expanded }) => (
```

Comment:
I don't know if this makes sense, but do you think that expanded could be part of a context?. I see a lot of expanded passing around

Reply:
I gave it a try and it didn't make the code easier to read/maintain